#float[] duration 

---
float32 mean      # Did it achieve it?